views/home.erb

```diff
@@ -5,6 +5,15 @@
 <h1 class="dcf-txt-h3">Welcome to Innovation Studio Manager</h1>
 
 <h5>Hello, <%= @user.full_name %> (<%= @user.username %>)</h5>
+<% if @user.expiration_date > Date.today%>
+    <% if @user.expiration_date > (Time.now + (2*7*24*60*60)) %>
```

Comment:
Instead of using magic numbers here with that ```2*7*24*60*60``` you should try to use the Integer class in language.rb to set it to 14 days

Reply:
Done.